bot/KPI.py

```diff
@@ -0,0 +1,64 @@
+import asyncio
+
+from aiogram import Bot, Dispatcher, types, Router, F
+from aiogram.filters import Command
+from aiogram.types import Message
+from aiogram.types import FSInputFile
+import httplib2
+from aiogram.fsm.context import FSMContext
+from aiogram.fsm.state import State, StatesGroup
+from aiogram.fsm.storage.memory import MemoryStorage
+from aiogram.filters import Command
+from googleapiclient.discovery import build
+from oauth2client.service_account import ServiceAccountCredentials
+import creds
+import logging
+import re
+from datetime import datetime
+
+BOT_TOKEN = ''
+# Создаем объекты бота и диспетчера
+bot = Bot(token=BOT_TOKEN)
+dp = Dispatcher()
+def get_service_sacc():
+    creds_json = "kpiinfo-f2eb171a014e.json"

```

Comment:
What is a source for this file? Is it safe to include it into public access of repository? Got a mistake: "FileNotFoundError: [Errno 2] No such file or directory: 'kpiinfo-f2eb171a014e.json'"

Reply:
@egor-cpu I have solved this in the new version of the file